demisto_sdk/commands/common/tools.py

```diff
@@ -2008,11 +2008,43 @@ def get_latest_upload_flow_commit_hash() -> str:
     return last_commit
 
 
-def get_content_path() -> Path:
+def find_pack_folder(path: Path) -> Path:
+    """
+    >>> find_pack_folder(Path('root/Packs/MyPack/Integrations/MyIntegration/MyIntegration.yml'))
```

Comment:
Please add this comment as tests/ add a link to where these are run

Reply:
https://docs.pytest.org/en/7.1.x/how-to/doctest.html